src/core/internal/convert.d

```diff
@@ -183,14 +181,15 @@ const(ubyte)[] toUbyte(T)(const ref T val) if (is(Unqual!T == float) || is(Unqua
 }
 
 @safe pure nothrow @nogc
-private Float parse(bool is_denormalized = false, T)(T x) if (is(Unqual!T == ifloat) || is(Unqual!T == idouble) || is(Unqual!T == ireal))
+private Float parse(bool is_denormalized = false, T:ireal)(T x)
```

Comment:
Needs a `__traits(isFloating)` check to keep the current semantics.

Reply:
Doesn't need it because `parse` is private and only called for 80 bit real / quadruple precision real.